### DISCUSSION TITLE
Exclude path from runtime caching

### DISCUSSION THREAD

**Discussion by kirkegaard** [DuCanhGH/next-pwa#57] [Q&A]
Id love to extend the default runtime cache but exclude a few specific paths. I never want them to hit cache at all. Is there a way of doing so? I tried something like this but it seems to be hitting the cache still.

```js
  extendDefaultRuntimeCaching: true,
  workboxOptions: {
    runtimeCaching: [
      {
        urlPattern: ({ url: { pathname } }) =>
          pathname.startsWith("/api/broker/") &&
          pathname.startsWith("/api/payment/")
        handler: "NetworkOnly",
        options: {
          cacheName: "excludes",
        },
      },
    ],
```

**Comment by DuCanhGH** [comment]
It seems that the response isn't cached for me. You may see `Size: (ServiceWorker)` when you look at the Network tab, but it's just the service worker intercepting the request to handle it. So I think it's working properly, though you will have to force Next to not cache it (if the header `X-Nextjs-Cache` is `HIT` then most likely it is cached). Perhaps you just need to add `export const dynamic = 'force-dynamic'`.

I hope this helps! I'm really sorry that you had to wait this long for a response, but I don't really check the `Discussions` tab and it doesn't help that I've been really busy lately :(

**Comment by khuezy** [comment]
@kirkegaard 
Shouldn't it be an OR? eg:
```
pathname.startsWith("/api/broker/") || pathname.startsWith("/api/payment/")
```
That is, if the path is `/api/broke/` or `/api/payment`, then it should never be cached.
Or am I missing something?
